app/src/main/java/tech/gruppone/stalker/app/business/Organization.java

```diff
@@ -31,6 +30,7 @@ public Organization(@NonNull JSONObject jsonOrg) {
       description = data.getString("description");
       isPrivate = data.getBoolean("isPrivate");
       places = new ArrayList<>();
+      connected = false;
```

Comment:
Non sono convintissimo che sia il posto migliore dove tenere questa roba qui. Non so come sia fatto il resto del codice, ma non vai più comodo ad avere ad es. un HashSet<Organization> connectedOrganizations?

Reply:
Per fare come dici tu bisognerebbe ristrutturare completamente il modo in cui passiamo avanti e indietro le informazioni, e non so quanto ne valga la pena.